packages/cli/src/commands/generate/scaffold/scaffold.js

```diff
@@ -436,6 +437,7 @@ const modelRelatedVariables = (model) => {
     editableColumns,
     listFormattersImports,
     formattersImports,
+    relations,
```

Comment:
adds the connivence of having the relations info available in the template, so the user can access it.